src/components/Footer.astro

```diff
@@ -0,0 +1,57 @@
+---
+import { Icon } from 'astro-icon';
+import { t } from 'i18next';
+
+const socials = [
+  {
+    name: 'Facebook',
+    href: 'https://facebook.com/GermanITSkills',
+    icon: 'ic:baseline-facebook',
+  },
+  {
+    name: 'Instagram',
+    href: 'https://instagram.com/german.it.skills',
+    icon: 'mdi:instagram',
+  },
+  {
+    name: 'YouTube',
+    href: 'https://youtube.de/@GermanITSkills',
+    icon: 'bi:youtube',
+  },
+];
+---
+
+<footer class="bg-gray-800">
+  <div
+    class="mx-auto max-w-7xl overflow-hidden py-10 px-4 sm:px-6 lg:px-8 md:flex md:items-center md:justify-between"
+  >
+    <nav class="-mx-5 -my-2 flex flex-wrap justify-center" aria-label="Footer">
+      <div class="px-5 py-2">
+        <a href="#" class="text-base text-gray-400 hover:text-gray-100"
+          >{t('layout.footer.imprint')}</a
+        >
+      </div>
+
+      <div class="px-5 py-2">
+        <a href="#" class="text-base text-gray-400 hover:text-gray-100"
+          >{t('layout.footer.privacy')}</a
+        >
+      </div>
```

Comment:
Map those links too (like socials)?

Reply:
It shouldn't be necessary since we will likely never change them. Socials, however, could be changed more frequently hence the map.

Reply:
I would have done this just for DRY, but it's ok